testsuite/features/step_definitions/command_steps.rb

```diff
@@ -593,6 +593,11 @@
   output, _code = $server.run('/usr/sbin/rhn-search cleanindex', check_errors: false)
   log "Search reindex finished." if output.include?("Index files have been deleted and database has been cleaned up, ready to reindex")
   raise 'The output includes an error log' if output.include?('ERROR')
+  repeat_until_timeout(timeout: 5000, message: "waiting for search index to become active again") do
```

Comment:
5000? AHAHAH!
No, no, please let's keep a very small value as maximum time, for instance 10 seconds.

Reply:
I thought it was in milisseconds :upside_down_face: it was supposed to be 5 seconds :sweat_smile: